packages/core/src/main/logger/file-transport.injectable.ts

```diff
@@ -8,7 +8,7 @@ import directoryForLogsInjectable from "../../common/app-paths/directory-for-log
 import { loggerTransportInjectionToken } from "../../common/logger/transports";
 
 const fileLoggerTranportInjectable = getInjectable({
```

Comment:
nit: can you also fix the injectable name

Reply:
Thanks, that typo seems to be hard to spot